docs/src/main/sphinx/release/release-456.md

```diff
@@ -0,0 +1,64 @@
+# Release 456 (4 Sep 2024)
+
+## General
+
+* Expose additional JMX metrics about resource groups, including CPU and memory
+  usage, limits, and scheduling policy. ({issue}`22957`)
+* Add [adaptive plan optimization](/optimizer/adaptive-plan-optimizations) which
+  can reorder joins at runtime. The feature can be disabled with the
+  `fault-tolerant-execution-adaptive-join-reordering-enabled` configuration
+  property or the `fault_tolerant_execution_adaptive_join_reordering_enabled`
+  session property. ({issue}`23046`)
+
+## JDBC driver
+
+* Add publishing a JDBC driver JAR without shaded, third-party dependencies. ({issue}`22098`)
+
+## BigQuery connector
+
+* Fix failures with queries using table functions when `parent-project-id` is
+  defined. ({issue}`23041`)
+
+## Blackhole connector
+
+* Add support for replacing tables. ({issue}`23004`)
+
+## Delta Lake connector
+
+* Add support for creating tables with deletion vectors. ({issue}`22104`)
```

Comment:
Is the concept of “deletion vector” visible to users? If it’s a feature in Delta Lake that users have to be aware of, add a link to their docs.

Reply:
Linking like we do in older release notes now.. maybe we need to add a docs section at some stage @findinpath @jhlodin